apps/health_campaign_field_worker_app/pubspec.yaml

```diff
@@ -92,6 +99,7 @@ dev_dependencies:
   isar_generator: ^3.0.5
   flutter_launcher_icons: ^0.13.1
   hrk_flutter_test_batteries: ^0.8.0
+  change_app_package_name: ^1.3.0
```

Comment:
This should be removed since its not being used.

Reply:
> This should be removed since its not being used.

removed